assets/chat/js/const.js

```diff
@@ -59,7 +59,8 @@ const GENERIFY_OPTIONS = {
     // 'dank': 'generify-dank',
     // 'hyper': 'generify-hyper generify-rustle',
     'love': 'generify-love',
-    'spin': 'generify-spin'
+    'spin': 'generify-spin',
+	'wide': 'wide'
```

Comment:
Needs tabbing over to match

Reply:
Un-resolving this. Should be 4-space indent to match the rest of the file.